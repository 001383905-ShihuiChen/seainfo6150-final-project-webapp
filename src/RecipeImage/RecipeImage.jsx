import React from 'react'
import css from "./RecipeImage.module.css";

const RecipeImage = ({url, text}) => {
  return (
    <div className = {css.bookCoverStyle}>
      <img src={url} alt={text} />
    </div>
  );
};

export default RecipeImage;