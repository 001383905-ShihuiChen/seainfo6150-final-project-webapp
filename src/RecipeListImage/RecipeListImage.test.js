import React from "react";
import { render } from "@testing-library/react";
import RecipeImage from "./RecipeListImage.jsx";

it("renders the RecipeListImage component correctly", () => {
  const { container } = render(
    <RecipeImage
      url="https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fimages.media-allrecipes.com%2Fuserphotos%2F4536480.jpg&w=596&h=596&c=sc&poi=face&q=85"
      
    />
  );
  expect(container).toMatchSnapshot();
});