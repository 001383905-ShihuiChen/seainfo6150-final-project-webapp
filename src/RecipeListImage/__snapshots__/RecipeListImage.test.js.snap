// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the RecipeImage component correctly 1`] = `
<div>
  <div
    class="imageStyle"
  >
    <img
      src="https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fimages.media-allrecipes.com%2Fuserphotos%2F4536480.jpg&w=596&h=596&c=sc&poi=face&q=85"
    />
  </div>
</div>
`;

exports[`renders the RecipeListImage component correctly 1`] = `
<div>
  <div
    class="imageStyle"
  >
    <img
      src="https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fimages.media-allrecipes.com%2Fuserphotos%2F4536480.jpg&w=596&h=596&c=sc&poi=face&q=85"
    />
  </div>
</div>
`;
